src/components/Images/BlankImage.tsx

```diff
@@ -0,0 +1,175 @@
+            fill="#91AFAC"
+          ></path>
+          <path
+            d="M126.565731,105.812618 C125.474458,105.812618 124.449222,105.402005 123.627052,104.624175 C122.707005,103.753892 122.172571,102.526061 122.265731,101.496816 C122.398986,100.019693 124.11408,99.1991745 128.38342,98.5694575 C130.432005,98.2673349 132.437429,98.1145047 133.181061,98.064033 C136.59191,95.8482311 144.938373,85.6399764 149.468797,80.0989387 C150.694505,78.5998821 151.662429,77.4161557 152.276344,76.7005896 C152.884363,75.9916274 154.572807,71.9996462 156.974222,64.6795991 C158.994269,58.5208726 160.397335,53.4067217 160.440495,52.4942217 C160.471156,51.84375 161.031769,51.3399764 161.673514,51.3715802 C162.324222,51.4020047 162.826816,51.9543632 162.796156,52.6045991 C162.667146,55.3487028 156.252052,75.6878538 154.066439,78.235967 C153.470684,78.9305425 152.510542,80.1048349 151.294976,81.5918632 C144.749458,89.5972877 137.486014,98.3005896 134.140731,100.242099 C133.980825,100.334788 133.80158,100.388797 133.617146,100.39941 C129.553231,100.635731 125.609599,101.309788 124.638137,101.914033 C124.700637,102.172052 124.885542,102.568042 125.248042,102.910731 C125.564788,103.210495 126.10441,103.544222 126.87375,103.431014 C128.248514,103.229363 129.051816,102.777948 129.059835,102.773231 C129.622807,102.452948 130.342618,102.643042 130.667854,103.203184 C130.99309,103.76309 130.810071,104.478184 130.252524,104.807901 C130.135071,104.877476 129.050165,105.495401 127.215967,105.764741 C126.997335,105.796816 126.780354,105.812618 126.565731,105.812618"
+            fill="#91AFAC"
+          ></path>
+        </g>
+      </g>
+    </Svg>
+  </Wrapper>
+)
+
+export const BlankImage = withTheme(BlankImageComponent)
+
+const Wrapper = styled.figure`
+  display: inline-block;
+  margin: 0;
+  padding: 0;
+`
+
+const Svg = styled.svg`
+  ${({ theme }: InjectedProps) => {
+    return css`
+      display: block;
+
+      &.s {
+        width: ${theme.size.pxToRem(166)};
+        height: ${theme.size.pxToRem(166)};
+      }
+
+      &.m {
+        width: ${theme.size.pxToRem(250)};
+        height: ${theme.size.pxToRem(250)};
+      }
+
+      &.l {
+        width: ${theme.size.pxToRem(375)};
+        height: ${theme.size.pxToRem(375)};
```

Comment:
[IMO] Since `theme.size.pxToRem` appears frequently, it is easy to see if pxToRem is made variable first.

```typescript
const Svg = styled.svg`
  ${({ theme }: InjectedProps) => {
    const { pxToRem } = theme.size
    return css`
      display: block;

      &.s {
        width: ${pxToRem(166)};
        height: ${pxToRem(166)};
      }

      &.m {
        width: ${pxToRem(250)};
        height: ${pxToRem(250)};
      }

      &.l {
        width: ${pxToRem(375)};
        height: ${pxToRem(375)};
      }
    `
  }}
`
```